lib/eigensolve_quda.cpp

```diff
@@ -337,11 +337,8 @@ namespace quda
     // C_1 is the current 'out' vector.
 
     // Clone 'in' to two temporary vectors.
-    ColorSpinorField *tmp1 = ColorSpinorField::Create(in);
-    ColorSpinorField *tmp2 = ColorSpinorField::Create(in);
-
-    blas::copy(*tmp1, in);
-    blas::copy(*tmp2, out);
+    auto tmp1 = std::make_unique<ColorSpinorField>(in);
```

Comment:
stack?

Reply:
I tried this initially, but the present of the `std::swap` makes this less trivial.  Since the point of this PR is not to clean up the solvers completely, I figured this could be deferred.